.github/autobuild/mac.sh

```diff
@@ -101,11 +107,11 @@ prepare_signing() {
     # Set up a keychain for the build:
     security create-keychain -p "${KEYCHAIN_PASSWORD}" build.keychain
     security default-keychain -s build.keychain
-    # Remove default re-lock timeout to avoid codesign hangs:
+    # # Remove default re-lock timeout to avoid codesign hangs:
```

Comment:
```suggestion
    ## Remove default re-lock timeout to avoid codesign hangs:
```

Reply:
Should remove the extra `#` too. I suspect that line was an accidental change.